install.sh

```diff
@@ -436,10 +439,57 @@ if [ $upgrade == true ]; then
         rsync -rlv --fuzzy --delay-updates --delete-delay \
             --exclude "logs" --exclude "documents" --exclude "migrations" --exclude "__pycache__" \
             README.md LICENSE conf $REQUIRED_MODULES $INSTALL_PATH/
+        
+        PROJECT_FOLDER="$INSTALL_PATH/$PROJECT_FOLDER"
+        if [ ! -f "$PROJECT_FOLDER" ]; then
+            # Starting Relecov Platform
+            echo "No valid $PROJECT_NAME project was found in $INSTALL_PATH. Creating it..."
+            cd $INSTALL_PATH
+
+            echo "activate the virtualenv"
+            source virtualenv/bin/activate
+
+            django-admin startproject "$PROJECT_NAME" .
+            if [ $? -ne 0 ]; then
+                echo "Error: Failed to create Django project. Aborting."
+                exit 1
+            fi
             
-        # update the settings.py and the main urls
-        echo "Update settings and url file."
-        update_settings_and_urls
+            # update the settings.py and the main urls
+            echo "Update settings and url file."
+            update_settings_and_urls
+
+            if [ $docker == false ]; then
+                echo "Creating the database structure for $PROJECT_NAME"
+                python manage.py migrate
```

Comment:
in upgrade i wouldn't do this, if you do upgrade you should already have your compatible database, and later in the code for upgrade you have the makemigrations and migrate. This could generate inconsistencies.